llama_stack/distribution/server/server.py

```diff
@@ -381,11 +395,36 @@ def main():
 
     import uvicorn
 
-    # FYI this does not do hot-reloads
+    # Configure SSL if certificates are provided
+    port = args.port or config.server.port
+
+    ssl_config = None
+    if args.ssl_keyfile:
+        keyfile = args.ssl_keyfile
+        certfile = args.ssl_certfile
+    else:
+        keyfile = config.server.ssl_keyfile
+        certfile = config.server.ssl_certfile
```

Comment:
Consider making this more Pythonic

Reply:
hm, how? this?
```python
keyfile, certfile = (args.ssl_keyfile, args.ssl_certfile) if args.ssl_keyfile else (...)
```

seems equally verbose

Reply:
Alright your current code is more readable